CORE_AND_ELECTIVE_COURSES/README.md

```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
 
 The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level-up each participant's understanding so that we all share a great baseline to build upon. The Elective Courses will cover even more bleeding-edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge and deepen their understanding.
 
-The courses list is: `TBA`
+The courses list is:
+
+|                | Title  | Trainers  | Short Description  | Link  |
+|--------------- |------- |---------- |------------------- |------ |
+| **Core**       |        |           |                    |       |
+| A              | Understanding how the InterPlanetary File System deals with Files       |           |                    | [CORE_COURSE_A](CORE_COURSE_A) |
+| B              | Processing Addressing wut? A packet epic adventure through the Internet       |           |                    | [CORE_COURSE_B](CORE_COURSE_B) |
+
+| C              | Developing Apps with IPFS. Pick implementation and way of interacting with it       |           |                    | [CORE_COURSE_C](CORE_COURSE_C) |
+
+| D              | The lifecycle of data in DWeb.       |           | Providing (Share), Pinning (Store) and Blocking (Delete) Data. How IPFS answers all of these questions.                   | [CORE_COURSE_D](CORE_COURSE_D) |
+
+| **Electives**  |        |           |                    |       |
+| A              | Identity on the DWeb       |           |                    | [ELECTIVE_COURSE_A](ELECTIVE_COURSE_A) |
+
+| B              | Building with IPFS Cluster       |           |                    | [ELECTIVE_COURSE_B](ELECTIVE_COURSE_B) |

```

Comment:
@hsanjuan @lanzafame @meiqimichelle, can you confirm that the course Title is still accurate?